src/lib/interfaces.ts

```diff
@@ -44,6 +44,23 @@ export interface RawQuizzes {
   [key: string]: RawQuiz
 }
 
+/**
+ * Table of Contents
+ */
+export interface ToCItem {
```

Comment:
This may be a remnant of trying to migrate this over to `types` as it was previously in `interfaces`... Agree though, per our latest guidelines we should switch to the `types.ts` instance and remove this

Reply:
Ah ok. Yes, we should try to have just one definition.